CyclopsNuclearReactor/CyNukeChargeManager.cs

```diff
@@ -1,16 +1,14 @@
 ﻿namespace CyclopsNuclearReactor
 {
-    using Common;
-    using MoreCyclopsUpgrades.CyclopsUpgrades.CyclopsCharging;
     using System.Collections.Generic;
+    using Common;
+    using MoreCyclopsUpgrades.API;
     using UnityEngine;
 
     internal class CyNukeChargeManager : ICyclopsCharger
     {
         #region Static Methods
 
-        // This system was modeled after the CyclopsManager of MoreCyclopsUpgrades given its proven reliability
-
         internal static readonly List<CyNukeChargeManager> Managers = new List<CyNukeChargeManager>();
```

Comment:
Change to use MCUServices for this instead of creating its own base level manager.